src/quo/components/buttons/wallet_ctas/view.cljs

```diff
@@ -15,9 +15,8 @@
    {:style               style/button-container
     :accessibility-label accessibility-label}
    [wallet-button/view
-    {:icon          icon
-     :on-press      on-press
-     :on-long-press #(js/alert "long pressed")}]
```

Comment:
unneeded on long press. In general we should try treat quo components as a final product and not add generic (redundant) info actions like this to these components